packages/common/src/stores.ts

```diff
@@ -1,5 +1,36 @@
+import { Level } from 'level';
+
 import type { KeyValueStore } from './types.js';
 
+export class LevelStore implements KeyValueStore<string, any> {
+  private store: Level<string, string>;
+
+  constructor(location = 'DATASTORE') {
+    this.store = new Level(location);
+  }
+
+  async clear(): Promise<void> {
+    await this.store.clear();
+  }
+
+  async close(): Promise<void> {
+    await this.store.close();
+  }
+
+  async delete(key: string): Promise<boolean> {
+    await this.store.del(key);
+    return true;
+  }
+
+  async get(key: string): Promise<any> {
+    return await this.store.get(key);
+  }
+
+  async set(key: string, value: any): Promise<void> {
+    await this.store.put(key, value);
+  }
+}
```

Comment:
React Native will need to to be able to provide a different `AbstractLevel` instance for the `store` here. The `Level` type currently used for `store` is for Node.js and browsers.